### PR TITLE
Add balrog

### PR DESCRIPTION
**Pull request by Markusjsommer**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by dpryan79** [comment]
@mbargull How should we handle pytorch packages? There's an old pytorch-cpu in conda-forge but there are newer GPU-enabled versions in the defaults and pytorch channels. I wonder if we/conda-forge should work together with the pytorch folks so they use the conda-forge pinnings and we can then add the pytorch channel into the mix.

**Comment by mbargull** [comment]
Isuru is working on GPU-enabled PyTorch packages. A couple of days ago he and Mark Harfouche updated the CPU version.
I can't tell how long it's going to take to get the GPU one build; annoyingly, it takes so long to build that the CI times out...
cf: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/22 and  https://github.com/conda-forge/pytorch-cpu-feedstock/issues/7#issuecomment-744169007 ff

**Comment by dpryan79** [comment]
Let's hold-off on balrog until that's resolved if possible then.

File: recipes/balrog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - pytorch-cpu >=1.5.0
+    - python
+
+test:
+  commands:
+    - balrog --help
+
+about:
+  home: https://github.com/Markusjsommer/BalrogCPP
+  license: MIT
```
Comment (mbargull) [inline recipes/balrog/meta.yaml:38]: @Markusjsommer, could you include a license file upstream which we could then add as a reference here with, e.g.,
```suggestion
  license: MIT
  license_file: LICENSE
```

Comment (Markusjsommer) [reply recipes/balrog/meta.yaml:38]: Added!

Comment (mbargull) [reply recipes/balrog/meta.yaml:38]: Perfect, thanks!

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

**Review by mbargull — APPROVED** [review]
LGTM, thanks for the contribution!

**Comment by mbargull** [comment]
@BiocondaBot, please merge

**Pull request merged**